client/src/web_pages/Welcome_Components/signup.jsx

```diff
@@ -19,7 +19,7 @@ const SignUp = (props) => {
 
                 <button onClick={() => {props.onSignupSubmitClick(); props.grantUserAccess();}} className='Submit'> Submit</button>
 
-                <button className='Cancel' onClick={props.onClickSignUp}> Cancel</button>
```

Comment:
This button and functionality was working already. The added button wasn't needed.